build/commands/scripts/commands.js

```diff
@@ -124,7 +125,8 @@ program
   .option('--notarize', 'notarize the macOS app with Apple')
   .option('--target_os <target_os>', 'target OS')
   .option('--target_arch <target_arch>', 'target architecture')
-  .option('--target_apk_base <target_apk_base>', 'target Android OS apk (classic, modern, mono)', 'classic')
+  .option('--target_apk_aab_base <target_apk_base>', 'target Android OS level for apk or aab (classic, modern, mono)', 'classic')
```

Comment:
`target_android_os_level` or something to indicate this is Android only

Reply:
The name is update.